database.c

```diff
@@ -22,6 +22,9 @@ static pthread_mutex_t dblock;
 static bool db_set_counter(const unsigned int ID, const int value);
 static int db_get_FTL_property(const unsigned int ID);
 
+// defined in networktable.c
+extern bool unify_hwaddr(sqlite3 *db);
```

Comment:
Would this be better in a header?

Reply:
It cannot be written in `routines.h` as the datatype `sqlite3` is not known in this file. I'd like to refrain from including `sqlite3.h` in each file by adding it through `routines.h`.